client/ayon_core/hosts/max/plugins/publish/validate_model_name.py

```diff
@@ -100,11 +100,12 @@ def invalid_name(cls, instance, obj):
             return obj
 
         # Validate regex groups
+        cls.log.debug(instance.data["productName"])
         invalid = False
         compare = {
             "project": instance.context.data["projectName"],
             "asset": instance.context.data["folderPath"],
```

Comment:
I suspect these should actually also be using `instance.data` as opposed to `context.data`? Depends on the use case of the validator.

Reply:
I dont think instance.data include ["projectName"] as I tested:
```
Traceback (most recent call last):
  File "C:\Users\Kayla\AppData\Local\Ynput\AYON\dependency_packages\ayon_2402141620_windows.zip\dependencies\pyblish\plugin.py", line 527, in __explicit_process
    runner(*args)
  File "D:\ayon-core\client\ayon_core\hosts\max\plugins\publish\validate_model_name.py", line 51, in process
    invalid = self.get_invalid(instance)
  File "D:\ayon-core\client\ayon_core\hosts\max\plugins\publish\validate_model_name.py", line 77, in get_invalid
    if cls.invalid_name(instance, obj):
  File "D:\ayon-core\client\ayon_core\hosts\max\plugins\publish\validate_model_name.py", line 103, in invalid_name
    cls.log.debug(instance.data["projectName"])
KeyError: 'projectName'

```

Reply:
Ah yes, project name is always on the context since it can't be customized on the instance - true. It should likely be `folderPath` only then. My bad.